libsplinter/src/transport/mod.rs

```diff
@@ -107,6 +107,10 @@ pub trait Transport: Send {
     fn accepts(&self, address: &str) -> bool;
     fn connect(&mut self, endpoint: &str) -> Result<Box<dyn Connection>, ConnectError>;
     fn listen(&mut self, bind: &str) -> Result<Box<dyn Listener>, ListenError>;
+
+    fn listen_all(&mut self, binds: &[String]) -> Result<Vec<Box<dyn Listener>>, ListenError> {
```

Comment:
This should not be added as the it will not properly correlate ListenError and is unnecessary.

Reply:
Removed